fabfile.py

```diff
@@ -42,7 +42,12 @@
 
 from fabric.api import *
 from fabric.utils import abort
-import configparser, string, random, os
+# import configparser, string, random, os
```

Comment:
Don't leave the old code commented out in your commit. If the new code works in your test, remove the old code. 

The same applies in the the other two files in this commit. Let the commit history document what you removed/replaced.